### PR TITLE
Update solidity version from 0.6 to 0.8

### PR DESCRIPTION
**Pull request by vkgnosis**

### CODE REVIEW

File: examples/truffle/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/gnosis/ethcontract-rs",
   "devDependencies": {
     "@gnosis.pm/util-contracts": "^3.0.1",
-    "@openzeppelin/contracts": "^3.4.0",
+    "@openzeppelin/contracts": "4.0.0-beta.0",
```
Comment (vkgnosis) [inline examples/truffle/package.json:26]: I needed to do this for the new solidity version. For some reason upgrading this (I think used a command like `yarn upgrade openzeppelin/contracts@4.0.0-beta.0`) has caused all the changes in the yarn.lock file. Not sure what is going on there.

**Review by fleupold — APPROVED** [review]
LGTM

File: examples/truffle/yarn.lock [hunk 2]
```diff
@@ -3,25 +3,25 @@
 
 
 "@babel/helper-module-imports@^7.12.1":
-  version "7.12.5"
-  resolved "https://registry.yarnpkg.com/@babel/helper-module-imports/-/helper-module-imports-7.12.5.tgz#1bfc0229f794988f76ed0a4d4e90860850b54dfb"
-  integrity sha512-SR713Ogqg6++uexFRORf/+nPXMmWIn80TALu0uaFb+iQIUoR7bOC7zBWyzBs5b3tBBJXuyD0cRu1F15GyzjOWA==
+  version "7.12.13"
+  resolved "https://registry.npmjs.org/@babel/helper-module-imports/-/helper-module-imports-7.12.13.tgz"
```
Comment (nlordell) [inline examples/truffle/yarn.lock]: Weird, it looks like all `s/yarnpkg.com/npmjs.org/` for the resolved URLs for the packages. What version of Yarn are you using?

Comment (vkgnosis) [reply examples/truffle/yarn.lock]: I am on `1.22.10`. I redid the upgrade this time without using `yarn upgrade` which has fixed this.

**Review by nlordell — APPROVED** [review]

**Pull request merged**